db/migrate/20221103092412_add_num_comments_to_submission_analyses.rb

```diff
@@ -0,0 +1,11 @@
+class AddNumCommentsToSubmissionAnalyses < ActiveRecord::Migration[7.0]
+  def change
+    add_column :submission_analyses, :num_comments, :integer, limit: 1, default: 0, null: false
```

Comment:
I've assumed that we'll never have an analyzer with more than 255 comments :)

Reply:
Can we specify algorithm and lock pls.

I think this is right, but check it:
```suggestion
    execute "ALTER TABLE submission_analyses ADD COLUMN num_comments TINYINT DEFAULT 0 NOT NULL, ALGORITHM=INPLACE, LOCK=NONE"
```